srcpkgs/python3-execnet/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-execnet'
+pkgname=python3-execnet
+version=1.9.0
+revision=1
+wrksrc="${pkgname#python3-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-setuptools_scm"
+checkdepends="python3-pytest"
+short_desc="Distributed Python deployment and communication"
+maintainer="classabbyamp <void@placeviolette.net>"
+license="MIT"
+homepage="https://codespeak.net/execnet/"
```

Comment:
I don't think that's still the current homepage, https://execnet.readthedocs.io/en/latest/ has more up to date content, including their deprecation notice.

Reply:
hm, I'll use `.../stable/`, because that matches the version packaged (instead of a release/commit that isn't tagged)

Reply:
Even better :)